### ISSUE TITLE
Docs that would be useful for a startup

### ISSUE THREAD

**Issue by mdangear** [CommonAccord/Cmacc-Org#6]
- Agreement to hire consultant
- Agreement to hire employees
- Agreement to sell stock or give stock options
- NDA (mutual and non-mutual)
- MOU
- Generic contract (just the standard stuff with anything related to an actual deal into an appendix - scope, price, schedule)

**Comment by mdangear** [comment]
just to clarify, ideally we build a set of docs in the S/ directory to cover this scope. Docs from the S/ directory should reference sections from the S/ directory so that it provides a self contained base.

**Comment by HazardJ** [comment]
http://new.commonaccord.org/index.php?action=source&file=S/01/MyDocMatrix/Doc_v0.md

As they said of Bill Clinton - I can give it to you square and give it to
you round.

**Comment by mdangear** [comment]
I see the beginning of the official docs at CommonAccord, this is great.
Will take a look and try to update my docs to make as much use as possible
of the standard sections there.
Great step forward :-)

**Comment by HazardJ** [comment]
:+1:  :)
A suggestion - you, as a business principal - ("client" in lawyer lingo) - could be very (most?) helpful in working on requirements - and let lawyers ("us" from perspective of lawyers) work on satisfying them.  For instance, expanding on the list(s) of preferred documents at http://new.commonaccord.org/index.php?action=source&file=S/OurLegal/Form/List_v0.md

I can find what we've got and others can add their own.

Make sense?

**Comment by mdangear** [comment]
The list as it is is an excellent start for me. Next would be to have
public IDs in U that start showing who is using the system. I can put my
name there, but public info should be limited. And then private info only
for people I actually deal with in P. A bit of duplication here but still
good.
Which makes me think people in U could be signers of a manifest that states
that legal docs should be shared or something like this.

**Comment by HazardJ** [comment]
Agreed on ids in U/  I should probably rename the current U/ to Ux/ (because the ids are fake).  Let me think about this.  Anyone who wants an id can put their contact info on their own site or GitHub account.  That way, they control the info, not us.  They can use the template of any of the existing ids.  This can't be very deep info, but that's ok because that kind of stuff needs to be handled in an UMA-ish way, in a full automation system.

**Comment by HazardJ** [comment]
Have a couple more - including the text for a French temporary employment - CDD.
